airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -194,10 +194,20 @@ public void close() throws Exception {
       return;
     }
 
-    LOGGER.debug("Closing normalization process");
+    LOGGER.debug("Terminating normalization process {}...", process.pid());
     WorkerUtils.gentleClose(process, 1, TimeUnit.MINUTES);
-    if (process.isAlive() || process.exitValue() != 0) {
-      throw new WorkerException("Normalization process wasn't successful");
+
+    /*
+     * After attempting to close the process check the following:
+     *
+     * Did the process actually terminate? If "yes", did it do so nominally?
+     */
+    if (process.isAlive()) {
+      throw new WorkerException("Normalization process " + process.pid() + " wasn't successful");
```

Comment:
is it more accurate to say that we didn't successfully  terminate it?

Reply:
@edgao Definitely.  Will fix.

Reply:
Done